### DISCUSSION TITLE
Is this project still alive?

### DISCUSSION THREAD

**Discussion by evilbaschdi** [AutoFixture/AutoFixture#1382] [General]
Sry for that cheeky title but I wanted to know if there's still someone who cares about that project.

The last release has been puslished nearly two years ago, the last commit was back in july of 2022.
Also the "release/5.0.0" branch is some kind of outdated since the last accepted pull request - 19 still open - was accepted in may of 2022.

I'm also asking since NSubstitute 5.0 is not supported currently.

**Comment by aivascu** [comment]
Hi @evilbaschdi ,
Yes it's still alive, it's just I've not been able to allocate time for open source the last year due to personal issues.

Lately I started having some spare time that I can allocate for the project, and I was trying to pick things up from where I left off.
As you can imagine there's a lot to go through, as before I put away the project there were several big changes waiting to happen.

I have been testing the NSubstitute 5.0 integration this weekend. There were also a couple more changes that I wanted to release in this patch, but if wont manage to get them merged by EOD, then I'll release the update for NSubstitute 5.0 without them.

The PR for NSubstitute is open in #1383

**Reply by evilbaschdi** [reply]
Nice to hear! And all the best for your personal issues!
